spec/requests/api/arbitration_profile_spec.rb

```diff
@@ -39,18 +41,126 @@
       expect do
         run_post(arbitration_profiles_url, gen_request(:create, request_body))
       end.to change(ArbitrationProfile, :count).by(1)
+
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'supports arbitration_default creation with provider id' do
+      api_basic_authorize collection_action_identifier(:arbitration_profiles, :create)
+      provider = FactoryGirl.create(:ext_management_system)
+
+      expect do
+        run_post(arbitration_profiles_url, request_body.merge(:provider => {:id => provider.id}))
+      end.to change(ArbitrationProfile, :count).by(1)
+      expect(response).to have_http_status(:ok)
```

Comment:
Is the response body important at all, or as a consumer are you only interested in the success/failure?